src/components/Time.jsx

```diff
@@ -0,0 +1,18 @@
+
+
+
+
+function Time(props){
+    const time = props.time 
+    const minutes = Math.floor((time % 360000) / 6000) + 60*Math.floor(time / 360000)
+    const seconds = Math.floor((time % 6000) / 100) + 60*minutes
+    const milliseconds = time % 100 
+
+    return(
+        <>
+        {seconds}.{milliseconds}s
```

Comment:
Only keep the seconds. There is not need for the milliseconds. Or make it configurable.

Reply:
I have made it configurable in [commit 4cce533](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/4cce533732da341c200f375b6f998969a68ac15d) and updated the README respectively in [commit 21c9821](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/21c9821a5c36f6a88d55113293486671daa2a390)